Sources/ConsoleKit/Utilities/ConsoleLogger.swift

```diff
@@ -82,6 +82,20 @@ public struct ConsoleLogger: LogHandler {
     }
 }
 
+private extension Logger.Metadata {
+    struct StringWithoutQuotes: CustomStringConvertible, Hashable {
+        let description: String
+    }
+
+    var descriptionWithoutQuotes: String {
+        var info: [StringWithoutQuotes: StringWithoutQuotes] = [:]
+        for (key, value) in self {
+            info[.init(description: key.description)] = .init(description: value.description)
+        }
+        return info.description
```

Comment:
```suggestion
        return self.reduce(into: [StringWithoutQuotes: StringWithoutQuotes]()) { description, pair in
            description[.init(description: pair.key.description)] = .init(description: pair.value.description)
        }.description
```

Reply:
I disagree with Caleb's suggestion. He has written it exactly as I would (nice to see him learning from the best 😉), but the result is not noticeably more readable and I don't believe it's any more efficient; I'd leave it as is.